src/colorizer/types.ts

```diff
@@ -167,3 +167,22 @@ export const defaultViewerConfig: ViewerConfig = {
   outOfRangeDrawSettings: { mode: DrawMode.USE_COLOR, color: new Color(OUT_OF_RANGE_COLOR_DEFAULT) },
   outlierDrawSettings: { mode: DrawMode.USE_COLOR, color: new Color(OUTLIER_COLOR_DEFAULT) },
 };
+
+export enum RangeType {
```

Comment:
Is this going to be another "permanent ids can never be changed" once you serialize this to url?

Reply:
We'll need to always ensure backwards compatibility for these specific values in the URL, but we can always add a layer of indirection here or remap them as needed.

Reply:
See here: https://github.com/allen-cell-animated/nucmorph-colorizer/pull/233/files#diff-49ae85ec948625f41a63acb8324ff556e601766969a1c26ac5c44c28c35ab61aR318-R328